### PR TITLE
Adding developer.wikimint.com web tools

### PR DESCRIPTION
**Pull request by selvaklnc**
[x] This is Software as a Service not self-hosted
[x] It has a free tier not just a free trial
[x] Pricing information is clearly visible without signup or phone calls
[x] The submission mentions what is free
[x] The submission is not already present in the list
[x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1905,6 +1905,7 @@ Update Time, five active automations, webhooks.
 
 ## Other Free Resources
 
+  * [Wikimint Developer](https://developer.wikimint.com/p/tools.html) - Always free tools for web developers that includes CSS minify unminify, image optimizer, image resizer, case convertor, CSS validator, JavaScript compiler, HTML editor, etc.
```
Comment (ripienaar) [inline README.md:1908]: Site keeps popping up the consent page on every page no matter how many times you press ok. Doesn't seem ready for general use.

Comment (selvaklnc) [reply README.md:1908]: Thanks for notifying the bug. Will fix it and inform you.

Comment (selvaklnc) [reply README.md:1908]: Hi @ripienaar, I've fixed the consent issue. Can you please kindly review it?

**Comment by ripienaar** [comment]
thanks

**Pull request merged**